### DISCUSSION TITLE
blog/daily/why-choose-nextjs-blog

### DISCUSSION THREAD

**Discussion by giscus** [deokgoo/next-duck-blog#8] [General]
# blog/daily/why-choose-nextjs-blog

blog는 역시 nextjs가 좋지 않을까?

https://duck-blog.vercel.app/blog/daily/why-choose-nextjs-blog

**Comment by ch1517** [comment]
정리된 글을보니 생각보다 SEO 말고도 이점이 많아서, 블로그를 직접 만들어 보고 싶다는 생각이 드는군요
영업왕 인정합니다!

**Comment by Mellbrown** [comment]
저도 한 블로그에 정착하지 못하고 게르만족 마냥, 지금 하신 전처를 그대로 밟고 있는데, 과연 제 미래인 거 같습니다! 글 아주 잘 봤습니다. 다음글도 빨리 올라오길 기대합니다. :)
